src/clj/chronograph/handlers/organization.clj

```diff
@@ -31,3 +35,33 @@
           response/response)
       (response/not-found
        {:error "Not found"}))))
+
+(defn invite
+  [{{:keys [slug]} :params
+    {user-id :users/id} :user
+    {:keys [email]} :body
+    :as _request}]
+  (jdbc/with-transaction [tx db/datasource]
+    (if-let [{org-id :organizations/id} (organization/find-by-slug tx slug)]
+      (if (s/valid? :invites/email email)
+        (if (acl/admin? tx user-id org-id)
+          (response/response
+           (invite/find-or-create! tx org-id email))
+          (-> (response/response {:error "Forbidden"})
+              (response/status 403)))
+        (response/bad-request {:error "Invalid email"}))
+      (response/not-found {:error "Not found"}))))
```

Comment:
I've been trying to keep these as "straight-line" as I can. Suppose we rewrite the body above as follows, would it read (and work) better?

```
(let [{org-id :organizations/id} (organization/find-by-slug tx slug)]
    (cond
      (not org-id)
      (response/not-found
       {:error "Not found"})

      (not (and (s/valid? :invites/email email)
                (s/valid? :organizations/slug slug)))
      (response/bad-request
       {:error "Invalid email or slug."})

      (not (acl/admin? tx user-id org-id))
      (-> (response/response {:error "Forbidden"})
          (response/status 403))

      :else
      (response/response
       (invite/find-or-create! tx org-id email))))```

Reply:
Done